app/src/main/java/ru/ft/giphyapp/di/AppModule.kt

```diff
@@ -2,9 +2,12 @@ package ru.ft.giphyapp.di
 
 import dagger.Module
 
-@Module
-interface AppModule {
-
-//    @Binds
-//    fun bindRepository(repositoryImpl: RepositoryImpl): Repository
-}
\ No newline at end of file
+@Module(
+    includes = [
+        NetworkModule::class,
+        RepositoryModule::class,
```

Comment:
Split by feature, not by package. Its normal to have `AppModule` + `NetworkModule` + `CoroutineModuel`